modules/minion/tests/minion/TaskTest.php

```diff
@@ -14,6 +14,48 @@
 
 class Minion_TaskTest extends KO7_Unittest_TestCase
 {
+	protected static $initial_request;
+	protected static $base_url;
+
+	/**
+	 * Sets up the environment
+	 */
+	// @codingStandardsIgnoreStart
+	public function setUp() : void
+	// @codingStandardsIgnoreEnd
+	{
+		parent::setUp();
+
+		KO7::$config->load('url')->set(
+			'trusted_hosts',
+			['www\.example\.com', 'www\.example2\.com']
+		);
```

Comment:
everytime you change some runtime configuration in the setup function, please do it via the setEnviroment function, because it creates a backup for the old config and restores it on tearDown.

Use something like:

````
$this->environmentDefault = [
	'url.trusted_hosts' => ['www\.example\.com', 'www\.example2\.com']
];
````

Reply:
also use this for other config changes in your setUp function